### ISSUE TITLE
NVDA Remote: Unable to generate key using Master

### ISSUE THREAD

**Issue by fisher729** [NVDARemote/NVDARemote#244]
Hi.

Using latest Master at the time of posting this comment:
1. Go to NVDA Remote's connect dialog.
2. In the host field, type nvdaremote.com
3. Tab to the generate key button.

Expected: The key is entered, and the field is moved to.
Actual: Nothing happens, and the following error is in the log:

ERROR - stderr (01:19:14.961) - Thread-31 (8304):
Exception in thread Thread-31:
Traceback (most recent call last):
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "C:\Users\username\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\transport.py", line 66, in run
    self.server_sock.connect(self.address)
  File "ssl.pyc", line 1172, in connect
  File "ssl.pyc", line 1163, in _real_connect
  File "ssl.pyc", line 1139, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)

What's happening here?

**Comment by tech10** [comment]
The server at nvdaremote.com likely has an invalid certificate which can't be verified, and thus, this fails. That's what this seems to indicate from the message.

Second, if the certificate verification is going to occur at the point of generating a key as well, there needs to be a method of properly showing the various dialogs required to verify a certificate, rather than trying to give the NVDARemoteServer at nvdaremote.com a valid certificate and calling it good.

If I'm wrong on both these counts, the issue is something else entirely.

**Comment by pvagner** [comment]
In the meantime nvdaremote.com appears to have valid certificate at least my system does trust it.
I have understood the issue and attempted to fix it with #252
